conf/references/mlss_conf.xml

```diff
@@ -440,6 +440,92 @@
       <genome name="daphnia_pulex"/>
       <genome name="agrilus_planipennis_gca000699045v2rs"/>
     </collection>
+
+    <collection name="liliopsida">
+      <base_collection name="shared"/>
+      <!-- Outgroups -->
+      <genome name="chlamydomonas_reinhardtii"/>
+      <genome name="marchantia_polymorpha"/>
+      <genome name="solanum_lycopersicum"/>
+      <!-- Asparagales -->
+      <genome name="asparagus_officinalis"/>
+      <!-- Chlamydomonadales -->
```

Comment:
Wouldn't any genome in`Chlamydomonadales` always be an outgroup genome w.r.t. Liliopsida?
```suggestion
```

Reply:
Forgot to remove that one, cheers!